### ISSUE TITLE
Accessibility issues.

### ISSUE THREAD

**Issue by EdijsApse** [wojtekmaj/react-daterange-picker#333]
1) There is no aria-expanded attribute for button which toggles calendar visibility.
2) NVDA Screen Reader doesn't read full date when navigating on dates. It reads instead only number.
3) Not an issue, but could be nice if user could navigate using arrow keys.
4) Also couldn't navigate with tab after going from year to month view and tried to select month. Calendar closes and needs to start over.

**Comment by EdijsApse** [comment]
You could add aria label with full date to button element instead of abbr tag. I did it in browser tools and screen reader was able to read full date.
![image](https://github.com/wojtekmaj/react-daterange-picker/assets/38587862/9594ca5a-5756-40c0-a78b-0ff31bc8a757)
